### DISCUSSION TITLE
Multi row tabs / tab wrapping

### DISCUSSION THREAD

**Discussion by virtuecoder** [caplin/FlexLayout#449] [Ideas]
Our customers request a possibility to wrap tabs into multiple rows when they cannot fit in one row. Do you have plans to support such a feature?

![Example](https://user-images.githubusercontent.com/27820864/104081455-efcb8d80-522e-11eb-807b-937780319eab.png)

**Comment by enjoysoft** [comment]
Feels a useful feature and commonly found in most tab ui framework.
